extensions/wikia/AdEngine/js/template/floating-rail.js

```diff
@@ -0,0 +1,101 @@
+/*global define*/
+define('ext.wikia.adEngine.template.floating-rail', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.uapContext',
+	'ext.wikia.adEngine.utils.math',
+	'ext.wikia.adEngine.adHelper',
+	'jquery',
+	'wikia.log',
+	'wikia.document',
+	'wikia.window'
+], function (adContext, uapContext, math, adHelper, $, log, doc, win) {
+	'use strict';
+
+	var $medrec = $('#TOP_RIGHT_BOXAD'),
+		$rail = $('#WikiaRail'),
+		$railWrapper = $('#WikiaRailWrapper'),
+		$wikiaMainContent = $('#WikiaMainContent'),
+		$win = $(win),
+		adsInRail = 2,
```

Comment:
adsInRail represents medrec and floating medrec? or what ads exactly are `adsInRail`?